rome-modules/src/main/java/com/rometools/modules/mediarss/io/MediaModuleParser.java

```diff
@@ -152,7 +165,12 @@ public Module parse(final Element mmRoot, final Locale locale) {
 
                     try {
                         if (content.getAttributeValue("fileSize") != null) {
-                            mc.setFileSize(Long.valueOf(content.getAttributeValue("fileSize")));
+                            String fileSizeAttr = content.getAttributeValue("fileSize").replaceAll("\\s", "");
```

Comment:
Moved to parseFileSize. Reckon the regex is complicated enough as it is though, and it's nice to be able to handle the case of whitespace in an otherwise numeric attribute value, so I left it as a seperate step in parseFileSize